spec/models/comment_spec.rb

```diff
@@ -1,5 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject { Comment.new(author_id: 1, post_id: 1, text: 'Lorem Ipsum is simply dummy text of the printing') }
+  describe 'Testing comment model to behave as expected' do
+    it 'Should create a comment with correct author_id' do
+      expect(subject.author_id).to eq(1)
+    end
+
+    it 'Should create a comment with correct post_id' do
+      expect(subject.post_id).to eq(1)
+    end
+
+    it 'Should create a comment with correct text' do
+      expect(subject.text).to be_kind_of(String)
+    end
+  end
```

Comment:
- [ ] 5 tests from 35 tests are failing, please fix them or if it is working on your end due to the different configuration, kindly attach the screenshot image that shows the passing tests.

![validate4](https://user-images.githubusercontent.com/55635977/147198013-b53787b4-7723-4001-a829-1bcd92162802.png)

Reply:
Yes, I had another bug there, now it's fixed and the suite it's passing as expected 👍 

Here is the screenshot:
![image](https://user-images.githubusercontent.com/82678907/147281777-51bcd18d-f8db-4685-a500-4e29e771b0e7.png)